.changeset/mighty-hairs-train.md

```diff
@@ -0,0 +1,61 @@
+---
+"@comet/admin": minor
+---
+
+Add a more actions menu helper component
+
+This is done to provide a simple api to create a more actions menu for a list of items. 
+
+**Example:**
+
+```tsx
+<MoreActionsMenu
```

Comment:
The implementation seems to have a few differences, compared to the design. 
Spacings and font-sizes for example. 

| Design | Implementation |
| --- | --- |
| <img width="269" alt="MoreActions Menu Design" src="https://github.com/vivid-planet/comet/assets/6264317/74a47a3e-9034-47ca-a05a-51f4bc54d7df"> | <img width="229" alt="MoreActions Menu Implementation" src="https://github.com/vivid-planet/comet/assets/6264317/d6ef5bab-78d7-4656-8e03-bd143e8dcfba"> |